test/e2e-v2/cases/profiling/ebpf/access_log/traffic-gen.yaml

```diff
@@ -33,8 +33,14 @@ spec:
     spec:
       containers:
         - name: trafficgenerator
-          image: elswork/wrk
-          command: ["wrk", "-t1", "-c1", "-d20m", "http://productpage:9080/productpage"]
```

Comment:
FYI @mrproliu , let’s avoid using such images that was not updated for 6 years and doesn’t work on Apple's M chip machines. Please consider removing this in satellite repo also.

Reply:
Yes, please update them after this gets merged.